migration-runner.sh

```diff
@@ -61,65 +44,67 @@ fi
 
 case "$ENV$OPERATION$DBTYPE" in
     localtakesnapshotsrealtime)
-        source steps/get-database-credentials.sh
         source steps/take-database-snapshots.sh
-#        source steps/start-local-postgres-server.sh
-#        source steps/load-database-snapshots.sh
-#        source steps/get-document-keys.sh
-#        source steps/export-document-keys.sh
-#        source steps/import-document-keys-to-data-store.sh
-#        source steps/migrate-staging-table.sh
-#        source steps/clean-up.sh
+        source steps/clean-up.sh
         ;;
     localloadsnapshotssnapshotdb)
-        source steps/get-database-credentials.sh
         source steps/start-local-postgres-server.sh
         source steps/load-database-snapshots.sh
-#        source steps/get-document-keys.sh
-#        source steps/export-document-keys.sh
-#        source steps/import-document-keys-to-data-store.sh
-#        source steps/migrate-staging-table.sh
-#        source steps/clean-up.sh
-        ;;
-    localexportdocumentidssnapshotdb)
-        source steps/get-database-credentials.sh
-        source steps/start-local-postgres-server.sh
-        source steps/get-document-keys.sh
-        source steps/export-document-keys.sh
-        source steps/import-document-keys-to-data-store.sh
-        source steps/migrate-staging-table.sh
-#        source steps/clean-up.sh
+        source steps/clean-up.sh
         ;;
     localexportrecursivedocumentidssnapshotdb)
-        source steps/get-database-credentials.sh
         source steps/start-local-postgres-server.sh
         source steps/recursive-staging-table.sh
-#        source steps/clean-up.sh
+        source steps/clean-up.sh
         ;;
     localexportrecursiveexceptionsnapshotdb)
-        source steps/get-database-credentials.sh
         source steps/start-local-postgres-server.sh
         source steps/recursive-exception-reports.sh
-#        source steps/clean-up.sh
+        source steps/clean-up.sh
         ;;
-    aatexportdocumentidsrealtime)
-        source steps/get-database-credentials.sh
-#        source steps/start-local-postgres-server.sh
-        source steps/get-document-keys.sh
-        source steps/export-document-keys.sh
-        source steps/import-document-keys-to-data-store.sh
-        source steps/migrate-staging-table.sh
-#        source steps/clean-up.sh
+    localexportrecursivedocumentidsrealtime)
+        source steps/recursive-staging-table.sh
+        source steps/clean-up.sh
         ;;
     aatexportrecursivedocumentidsrealtime)
-        source steps/get-database-credentials.sh
         source steps/recursive-staging-table.sh
-#        source steps/clean-up.sh
+        source steps/clean-up.sh
         ;;
     aatexportrecursiveexceptionrealtime)
-        source steps/get-database-credentials.sh
         source steps/recursive-exception-reports.sh
-#        source steps/clean-up.sh
+        source steps/clean-up.sh
+        ;;
+    demoexportrecursivedocumentidsrealtime)
```

Comment:
We could use the following `OR` syntax, but if anything it makes it more difficult to read (once it is expanded to cover all cases)


```
demoexportrecursivedocumentidsrealtime | ithcexportrecursivedocumentidsrealtime )
    source steps/recursive-staging-table.sh
    source steps/clean-up.sh
```

So will stick with some of this repetition for now